frontend/src/util/game_util.js

```diff
@@ -0,0 +1,5 @@
+export const getNewGame = () => axios.get('/api/games/new');
+
+export const patchGame = ({ gameId, guess, timeRemaining, timeElapsed }) => {
+  return axios.patch(`api/games/${gameId}`, { guess, timeRemaining, timeElapsed });
+}
```

Comment:
Need to import axios? Also see [the docs](https://github.com/axios/axios#global-axios-defaults) for attaching our bearer token